src/ai/backend/manager/data/permission/role.py

```diff
@@ -20,6 +20,7 @@ class RoleCreateInput:
 
     scope_permissions: list[ScopePermissionData] = field(default_factory=list)
     object_permissions: list[ObjectPermissionData] = field(default_factory=list)
+    id: Optional[uuid.UUID] = None
```

Comment:
It's better to receive the internally generated ID upon creation, rather than receiving an ID at the creation time.